hcl/printer/testdata/basic_list.golden

```diff
@@ -0,0 +1,13 @@
+listed_var = "tested var"
+bacon = "delicious"
+number = 3
+bool_var = true
```

Comment:
I don't think this is right - I think these equal signs should be aligned. 

Could you also add a test case for when the comment isn't on the left column:

```hcl
foo = [
  # comment
  "abc",

  # another comment
  "foo",
]
```